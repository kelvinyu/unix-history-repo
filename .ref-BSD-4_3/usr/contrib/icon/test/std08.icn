record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("\"x\" << \"x\" ----> ",image("x" << "x") | "none")
   write("\"x\" << \"X\" ----> ",image("x" << "X") | "none")
   write("\"X\" << \"x\" ----> ",image("X" << "x") | "none")
   write("\"xx\" <<= \"xx\" ----> ",image("xx" <<= "xx") | "none")
   write("\"xxx\" <<= \"xx\" ----> ",image("xxx" <<= "xx") | "none")
   write("\"xx\" <<= \"xxx\" ----> ",image("xx" <<= "xxx") | "none")
   write("\"x\" >>= \"x\" ----> ",image("x" >>= "x") | "none")
   write("\"x\" >>= \"xx\" ----> ",image("x" >>= "xx") | "none")
   write("\"xx\" >>= \"x\" ----> ",image("xx" >>= "x") | "none")
   write("\"x\" >> \"x\" ----> ",image("x" >> "x") | "none")
   write("\"x\" >> \"X\" ----> ",image("x" >> "X") | "none")
   write("\"X\" >> \"x\" ----> ",image("X" >> "x") | "none")
   write("\"x\" == \"x\" ----> ",image("x" == "x") | "none")
   write("\"x\" == \"X\" ----> ",image("x" == "X") | "none")
   write("\"X\" == \"x\" ----> ",image("X" == "x") | "none")
   write("\"x\" ~== \"x\" ----> ",image("x" ~== "x") | "none")
   write("\"x\" ~== \"X\" ----> ",image("x" ~== "X") | "none")
   write("\"X\" ~== \"x\" ----> ",image("X" ~== "x") | "none")
   write("[] ~== \"x\" ----> ",image([] ~== "x") | "none")
end
