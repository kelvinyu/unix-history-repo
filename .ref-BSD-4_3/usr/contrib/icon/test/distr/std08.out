"x" << "x" ----> none
"x" << "X" ----> none
"X" << "x" ----> "x"
"xx" <<= "xx" ----> "xx"
"xxx" <<= "xx" ----> none
"xx" <<= "xxx" ----> "xxx"
"x" >>= "x" ----> "x"
"x" >>= "xx" ----> none
"xx" >>= "x" ----> "x"
"x" >> "x" ----> none
"x" >> "X" ----> "X"
"X" >> "x" ----> none
"x" == "x" ----> "x"
"x" == "X" ----> none
"X" == "x" ----> none
"x" ~== "x" ----> none
"x" ~== "X" ----> "X"
"X" ~== "x" ----> "x"
Run-time error 103 at line 27 in std08.icn
string expected
offending value: []
